composer.json

```diff
@@ -21,15 +21,15 @@
         }
     },
     "require": {
-        "php": ">=7.2",
+        "php": ">=7.3",
```

Comment:
this change is not really required. yes, it's implied by the dependency, but the project itself does not require 7.3

Reply:
Onestly I don't know why and if respect/validation requires php 7.3 instead of 7.2, but its composer.json has this:
```
"php": "^7.3 || ^8.0",
```

Reply:
I just mean that changes in dependencies are not obligatory to make changes in project requirements. composer will resolve all minimul requirements by itself, so we can keep 7.2 here (since we may once change validator lib to another with another dependencies). 7.2 here means that the code of this library itself requires 7.2 to work, not for code of the dependencies

Reply:
You're right, it seems that composer lets the installation go ahead iven with the 7.2 constraint.
This looks strange to me, but maybe I misunderstood something, I wonder what would happen if you call methods in respect/validation that require php 7.3...

Reply:
Nothing. If this library require 7.2, respect/validation require 7.3 and you successfully installed this library - you using 7.3. Other cases are disallowed by composer. In that cases you will get only that versions of this libarry which have previous verions of respect/validation (with 7.2 for example).